### ISSUE TITLE
'col' is appended to all Column Elements

### ISSUE THREAD

**Issue by danman113** [reactstrap/reactstrap#848]
### Issue description

- components: `Col`
- reactstrap version `#5.0.0-beta`
- import method `es` 
- react version `#16.2.0`
- bootstrap version `#4.0.0`

### What is happening?

I'm just trying to have a `col-md-8` and `col-md-4` column, but the `Col` element is appending the `col` class to the element, causing the columns to become evenly spaced at smaller viewports: the opposite of what should be happening. Is there a way to specify not to append the `col` class?

### Code

Here's my jsx
```
<Container className='mt-5'>
  <Row>
    <Col md='8'>
      ....
    </Col>
    <Col md='4'>
      ....
```

**Comment by TheSharpieOne** [comment]
This is fixed via #842 but not yet released. For now the workaround is adding `xs={false}`

**Comment by danman113** [comment]
@TheSharpieOne Thanks, I'll use that for the time being!
